device/cli.py

```diff
@@ -22,12 +28,19 @@
 from common.tunnel import allocate_address_space, write_wireguard_config, start_wireguard_tunnel
 from common.models import TunnelRequest, TunnelRequestTokenData, Token, TunnelServerLaunchDetailsResponse, DeviceTunnelLaunchDetails, TunnelState
 
+config = configparser.ConfigParser()
+# TODO: only load files when they cannot be written by anybody other than `whoami`.
+config.read([
+    Path(os.path.dirname(__file__)) / "defaults.ini",
+    Path("/etc/support_tunnel/config.ini")
+])
+
```

Comment:
Should we add a wrapper for config['device'] to ensure this all works if the configuration is messed up for some reason? Something like:

```
def device_config() -> Dict:
  return config.get('device', {})
```

Reply:
or 
```
def device_config() -> Dict:
  try:
    return config['device']
  except KeyError as e:
    raise Exception("Unable to read device config") from e
```

Reply:
your first suggestion has a more pythonic expectation of the `configparser` API, but unfortunately it doesn't work like that - the "sections" are handled a smidge differently than a value that you can `.get()`. I'm also a little reluctant to raise an exception when we're unable to load configs, largely because we may see boxes with extremely broken filesystems and I'd like to tolerate that as much as possible.

I've made a new rendition of this that attempts to load the file, and should everything fail create an empty default, which I think still addresses your point about being "more robust" (but lemme know if it still smells.) This works because we specify a default for every `.get()` - perhaps a more brittle implementation for the future programmer, but imo we should be adding defaults for any `.get()`/env var anyways. I've also removed the merging of the `defaults.ini` and implemented a checker to prevent privilege escalation on both the scripts and the config file.